manifests/charts/istio-cni/values.yaml

```diff
@@ -47,6 +47,14 @@ cni:
     brokenPodLabelKey: "cni.istio.io/uninitialized"
     brokenPodLabelValue: "true"
 
+  resources:
```

Comment:
Should we use cilium's defaults?
```
$ helm template cilium cilium/cilium | k grep DaemonSet/ | grep resources: -A 5
        resources:
          requests:
            cpu: 100m
            memory: 100Mi
      restartPolicy: Always
      priorityClassName: system-node-critical
```

They are a more well established CNI, likely have those settings for a reason. I am concerned about setting low limits on memory like this. I suspect it will OOM on large cluster.

IMO setting limits by default is a bad idea

Reply:
Ok thanks for the suggestion! sgtm to remove limit.